src/Microsoft.Azure.SignalR.Common/Interfaces/IServiceConnectionContainer.cs

```diff
@@ -19,8 +20,16 @@ internal interface IServiceConnectionContainer
 
         Task<bool> WriteAckableMessageAsync(ServiceMessage serviceMessage, CancellationToken cancellationToken = default);
 
+        Task StartGetServersPingAsync();
+
+        Task StopGetServersPingAsync();
+
         ServiceConnectionStatus Status { get; }
 
         Task ConnectionInitializedTask { get; }
+
+        HashSet<string> GlobalServerIds { get; }
+
+        bool HasClients { get; }
```

Comment:
will these properties be used outside of the container? Is private propeties of `ContainerBase` enough?